src/components/SignUpPage/Form.js

```diff
@@ -0,0 +1,11 @@
+function Form(){
+  return(
+    <div className="w-[30%] flex flex-col justify-center items-center my-4" >
+      <input type="text" placeholder="Username"  className="bg-white border border-gray-300 text-grey-900 text-sm placeholder:text-placeholder-grey rounded-md focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 pl-5 mb-4 "/>
```

Comment:
Since this input is repeated, why don't you make it a component that has this style and takes in a param for the placeholder. That way it can be reused for the future as well.

Reply:
Created an input box component which takes a string for the placeholder.